feat(curriculum): adding content to css and accessibility review page (#57102)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/review-css-accessibility/671a955b74ab5588735800d1.md

Review the concepts below to prepare for the upcoming quiz.

## Color Contrast Tools

- **WebAIM's Color Contrast Checker**: This online tool allows you to input the foreground and background colors of your design and instantly see if they meet the Web Content Accessibility Guidelines (WCAG) standards. 
- **TPGi Colour Contrast Analyzer**: This is a free color contrast checker that allows you to check if your websites and apps meet the Web Content Accessibility Guidelines (WCAG) standards. This tools also comes with a Color Blindness Simulator feature which allows you to see what your website or app looks like for people with color vision issues.

## Best Practices With CSS and Accessibility    

- **`display: none;`**: Using `display: none;` means that screen readers and other assistive technologies won’t be able to access this content, as it is not included in the accessibility tree. Therefore, it is important to use this method only when you want to completely remove content from both visual presentation and accessibility.

> - **`visibility: hidden;`**: This property and value hides the content visually but keeps it in the document flow, meaning it still occupies space on the page. These elements will no longer be read by screen readers because it will have been removed from the accessibility tree. 

Reviewer: ```suggestion
- **`visibility: hidden;`**: This property and value hides the content visually but keeps it in the document flow, meaning it still occupies space on the page. These elements will also no longer be read by screen readers because they will have been removed from the accessibility tree. 
```

> - **`visibility: hidden;`**: This property and value hides the content visually but keeps it in the document flow, meaning it still occupies space on the page. Unlike `display: none;`, which removes content from the accessibility tree, `visibility: hidden;` does not remove content from the accessibility tree. This means that assistive technologies, like screen readers, can still access the hidden content.

Reviewer: This needs to be rewritten. `visibility: hidden` does remove the element from the accessibility tree.

[MDN: visibility](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility
)

Author: we could do something like this instead

```suggestion
- **`visibility: hidden;`**: This property and value hides the content visually but keeps it in the document flow, meaning it still occupies space on the page. These elements will no longer be read by screen readers because it will have been removed from the accessibility tree. 
```
